Userland/Libraries/LibWeb/Painting/StackingContext.cpp

```diff
@@ -544,10 +540,8 @@ Optional<HitTestResult> StackingContext::hit_test(CSSPixelPoint position, HitTes
     // 4. the non-positioned floats.
     for_each_in_subtree_of_type_within_same_stacking_context_in_reverse<PaintableBox>(paintable_box(), [&](PaintableBox const& paintable_box) {
         // FIXME: Support more overflow variations.
```

Comment:
Can these FIXMEs be removed now?

Reply:
(Same for the others, if not move this into the new function)

Reply:
I don't think these can be removed as we still have to handle `clip`, `scroll` and `auto`. I've moved the comment to the new function :)